website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -2,10 +2,201 @@
+
+- `Products`
+- `Categories`
+- `Customers`
+- `CustomerGroups`
+- `Orders`
+- `ProductDiscounts`
+- `CartDiscounts`
+- `DiscountCodes`
+- `ProjectSettings`
+- `ProductTypes`
+- `DeveloperSettings`
+
+## Action rights
+
+Action rights can be used to constrain a granted general permission. As manage access to products can be limited by not allowing publishing of products for a team.
+
+An action right is represented as an object with the shape `{ group: string, name: string }`. The `group` relates to the permission while the `name` is the action right itself. Currently the following action rights for the `group: 'products'` exist:
+
+- `PublishProducts`
+- `UnpublishProducts`
+- `AddPrices`
+- `EditPrices`
+- `DeletePrices`
+- `DeleteProducts`
+- `AddProducts`
+
+# Using configured permissions within a Custom Application
+
+The [`permissions`](https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/permissions/README.md)-package should be used to integrate configured permissions within a Custom Application. It offers React components, higher-order component and a hook.
+
+- `useIsAuthorized()`
+- `injectAuthorized()`
+- `<RestrictedByPermissions>`
+
+## `useIsAuthorized(options)`
+
+A React hook to evaluate passed in demanded permissions against the applied ones for the user's team.
+
+### Supported arguments
+
+- `demandedPermissions`: an array of `Permission`, requested for the child component to
+  be allowed to render
+- `demandedAtionRights`: (_optional_) an array of action rights
+
+```jsx
+import { PERMISSIONS } from './constants/permissions.js';
+
+const ExampleComponent = () => {
+  const canManageProducts = useIsAuthorized({
+    demandedPermissions: [PERMISSIONS.ManageProducts],
+  });
+
+  return <span>{ canManageProducts ? 'Authorized' : 'Not authorized' }</span>;
+};
+```
+
+## `injectAuthorized(permissions, [options])`
+
+A React higher-order component to inject the evaluation of demanded permission against the applied ones into a base component.
+
+### Supported arguments
+
+- `permissions`: an array of demanded general permissions
+- `options` (_optional_)
+  - `actionRights`: (_optional_) an array of action rights - if passed both, general permissions and action rights needs to be applied to the user's team
+
+```jsx
+const InputField = props => (
+  <Input
+    disabled={!props.isAuthorized}
+  />
+);
+
+export default injectAuthorized([PERMISSIONS.ViewProducts])(
+  InputField
+);
+```
+
+## `<RestrictedByPermissions>`
+
+A React component which renders its children if the demanded permissions
+match, otherwise the passed fallback component.
+
+### Supported props
+
+- `permissions`: an array of demanded general permissions
+- `actionRights`: (_optional_) an array of demanded action rights
+- `unauthorizedComponent`: (_optional_) a function return an React element to be
```

Comment:
```suggestion
- `unauthorizedComponent`: (_optional_) a React component instance to be rendered in case the evaluation of the permissions returns `false`.
```